### DISCUSSION TITLE
使用qiankuan配置微前端，我想要为配置子应用请求后端地址的host和主应用不同。应该在哪里配置请求拦截

### DISCUSSION THREAD

**Discussion by moosun123** [umijs/umi#10939] [Q&A]
使用qiankuan配置微前端，我想要为配置子应用请求后端地址的host和主应用不同。应该在哪里配置请求拦截。
我在主应用和子应用的拦截器 requestInterceptors 中配置了拦截， 都不能拦截到子应用的请求。

**Comment by fz6m** [comment]
就是 entry ，会按照 entry 的地址拉取 html ，你改成 `//domain.com` ，再区分下环境从不同的地方拉取就可以了
